CHANGELOG.md

```diff
@@ -7,6 +7,37 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes -->
 
+## [v0.92.0-sumo-0]
+
```

Comment:
Please add a link to the migration guide here.

Reply:
Added in this commit https://github.com/SumoLogic/sumologic-otel-collector/pull/1431/commits/89ff99b531f8a87d60eaa116150e72d00a6ed98f